gravity-forms/gw-redirect-if-email-exists.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Redirect if Email Exists
+ * https://gravitywiz.com/
+ * Redirect to a specified URL if the the submitted email address matches an existing user.
```

Comment:
There should be a space between these two lines. You can fix this directly on master though. No need for a separate PR.